package.json

```diff
@@ -58,16 +58,18 @@
   "dependencies": {
     "@improbable-eng/grpc-web": "^0.12.0",
     "@textile/threads-client-grpc": "^0.1.12",
+    "@textile/threads-core": "0.0.3-alpha.1",
     "@types/google-protobuf": "^3.7.2",
+    "@types/multiaddr": "^7.2.0",
```

Comment:
I just pushed another release that means you don't have to explicitly add these deps. But ok to leave them here for now.

Reply:
udpated